akka-javasdk/src/main/java/akka/javasdk/eventsourcedentity/ReplicationFilter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.eventsourcedentity;
+
+import akka.javasdk.impl.effect.ReplicationFilterImpl;
+
+public interface ReplicationFilter {
+  static ReplicationFilter empty() {
+    return ReplicationFilterImpl.empty();
+  }
+
+  static ReplicationFilter includeRegion(String region) {
+    return ReplicationFilterImpl.empty().addRegion(region);
+  }
+
+  static ReplicationFilter excludeRegion(String region) {
+    return ReplicationFilterImpl.empty().removeRegion(region);
+  }
+
+  ReplicationFilter addRegion(String region);
```

Comment:
would have been nice if the static method could have been named the same, but that's not possible with Java

Reply:
thanks, changed to something like that in https://github.com/akka/akka-sdk/pull/292/commits/ef93a539b9141e0c33fdfb83010394c47d324a95